recipes/backup_client.rb

```diff
@@ -17,10 +18,27 @@
 template "#{node['stash']['home_path']}/backup-config.properties" do
   source 'backup-config.properties.erb'
   owner node['stash']['user']
-  mode '0644'
-  variables :backup_client => settings['backup_client']
+  mode '0600'
+  variables :backup => settings['backup']
 end
 
 link "#{node['stash']['backup_client']['install_path']}/stash-backup-client/backup-config.properties" do
   to "#{node['stash']['home_path']}/backup-config.properties"
 end
+
+directory node.set['stash']['backup']['backup_path'] do
+  owner node['stash']['user']
+  mode '0750'
```

Comment:
or change to default not to use `/tmp`? create a folder elsewhere? I understand the reason behind you set it to `0750`. Other user on the servers should not be able to pull all your backup.